power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/domain/Procedure.java

```diff
@@ -0,0 +1,71 @@
+package com.adorsys.gis.powerpay.powerpaybackend.domain;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class Procedure {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    public   Status status = Status.WAITING;
+    private String phoneNumber;
+
+
+    public void markAsDone() {
+        status = Status.DONE;
+    }
+
+    public void markAsError() {
+        status = Status.ERROR;
+    }
+
+    public boolean isDone() {
+        return status == Status.DONE;
+    }
+
+    public boolean isError() {
+        return status == Status.ERROR;
+    }
+
+    public boolean isWaiting() {
+        return status == Status.WAITING;
+    }
+
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public Status getStatus() {
+        return status;
+    }
+
+    public void setStatus(Status status) {
+        this.status = status;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        this.phoneNumber = phoneNumber;
+    }
+
+    public enum Status {
```

Comment:
Put this in a separate file please

Reply:
In general i should put the two enumerations in a different file(same file for both enumerations ) ???

Reply:
Each enum should be in a different file

Reply:
ok sir

Reply:
ok sir

Reply:
Hello sir i just did some research on how to place enum's on spring boot application..

i notice i need to create a package called enum's then create two classes inside to put the "Status" and the "Type" enums

Reply:
You already have a package named domain. So you'll want to put the Type and Status class there. As there might be multiple other Status & Type in the future, it's better to use names like `ProcedureStatus`.

Reply:
ok sir, understood